driver/registry_default_hooks.go

```diff
@@ -73,3 +77,39 @@ func (m *RegistryDefault) hooksPost(credentialsType identity.CredentialsType, co
 
 	return i
 }
+
+func newRedirector(m *RegistryDefault, h configuration.SelfServiceHook, credentialsType identity.CredentialsType) (*hook.Redirector, error) {
```

Comment:
I think that this logic belongs to `NewRedirector` in hook package, but then we will couple it with config parsing. So I've left it there.

Reply:
Yeah I was about to say that! :D We could however replace `h configuration.SelfServiceHook` with just `config json.RawMessage` and replace the `WithField("hook", h.Run).` line with the "redirector" because we know what hook we're creating here :) What do you think?